README.md

```diff
@@ -1,24 +1,210 @@
+Gemme Fatale utilized polymorphic associations to have destinations be visitable association. The polymorphic association allowed countries to have destinations and the user is able to make new destinations (trips) without any country destinations being deleted. 
+
+**app/models/country.rb**
+```ruby
+class Country < ApplicationRecord
+    has_many :destinations, as: :visitable
+    validates :name, :prime_destination, :primary_language, :known_for, :climate, :nightlife, :image, presence: true
+end
+```
+**app/models/destination.rb**
+```ruby
+class Destination < ApplicationRecord
+    belongs_to :visitable, polymorphic: true
+end
+```
+**app/models/user.rb**
+```ruby
+class User < ApplicationRecord
+  devise :database_authenticatable, :registerable,
+  :recoverable, :rememberable, :validatable;
+  has_many :destinations, as: :visitable
+end
+```
+
+## 🚗 Testing
+To run the existing testing suite, run:
+
+```bash
+yarn jest
+rspec spec/
+```
+
+### Nomadda Data Specs
+Part of your responsibility will be to build out robust tests both for models and for requests. 
+Tests you will need are:  
+
+REQUEST: 
+- to ensure a user can see all apartments
+- to ensure a user can see all apartments that belong to them
+- to ensure a user can make a new apartment
+- to ensure a user can update an apartment
+- to ensure a user can remove an apartment
+- to ensure a user cannot make a new apartment with nil values
+- to ensure an error will be thrown if an unregistered user tried to make an apartment
+- to ensure an error will be thrown if a user tries to edit an apartment that doesn't belong to them
+- to ensure an error will be thrown if a user tries to delete an apartment that doesn't belong to them
+
+MODELS: 
+- to ensure destination is valid
+- to ensure a user cannot make a new apartment with nil values
+- to ensure a user cannot make a new apartment that already exists in the database
+- to ensure a user cannot update an apartment with nil values
+- to ensure a user cannot update another user's apartment
+
+**The following code is a mock and does not work but is here to get yours started.**
+```ruby
+require 'rails_helper'
+
+RSpec.describe "Destinations", type: :request do
+  describe "GET /index" do
+    it "returns a list of destinations" do
+      Destination.create(
+        location: "Waikiki"
+        main_attraction: "beauitful beaches"
+        start_date: '2023-01-01'
+        end_date: '2023-01-02'
+        travelers: 1
+        trip_details: "fbgm"
+        family_friendly: true
+        image: "https://images.unsplash.com/photo-1589394815804-964ed0be2eb5?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwcm9maWxlLWxpa2VkfDEzfHx8ZW58MHx8fHw%3D&auto=format&fit=crop&w=800&q=60"
+        visitable_id: 2
+        visitable_type: "User"
+        country: "Country"
+      )
+      get '/destinations'
+      destination = JSON.parse(response.body)
+      expect(response).to have_http_status (200)
+      expect(destination.length).to eq 1 
+    end
+  end
+```
```

Comment:
You can remove all of the configurations - basically lines 59 - 210.  Those should already be in your code. And your test should already run.  :)